### DISCUSSION TITLE
Where can I find a list of config.json options with their descriptions?

### DISCUSSION THREAD

**Discussion by Creogenesis** [Ylianst/MeshCentral#5762] [Q&A] [answered]
I remember seeing a page (website) where there was a complete list of options that can be set in **config.json**, but now I can't find it anywhere. Please, can someone point me to where to look for this list? 

I mean not just the enumerated options, but also with a brief description of these functions. I couldn't find a link on the main MeshCentral documentation site.

Because I really want to explore all the MeshCentral configuration options. I've found that you can achieve very flexible customization for yourself. For example, the option "manageAllDeviceGroups": ["user//admin"] can be intuitively understood for what it is, but it's also not possible to figure out many other options.

**Comment by elpibedeoro** [comment]
Hi,

There are 2 MeshCentral config schemas :

1. https://github.com/Ylianst/MeshCentral/blob/master/meshcentral-config-schema.json
2. https://github.com/Ylianst/MeshCentral/blob/master/sample-config-advanced.json

Hope this is what you asked for.

**Comment by si458** [comment, marked as answer]
all of the schema use to be displayed here - https://ylianst.github.io/MeshCentral/meshcentral/config/
HOWEVER the search box was goofing up, so we removed it, and replaced it with just pictures so people could have a rough guess at what the options are
we are still looking for something like a JSON schema editor that we could add into meshcentral to help people out
but as @elpibedeoro said the schema is here with everything including descriptions https://github.com/Ylianst/MeshCentral/blob/master/meshcentral-config-schema.json
https://ylianst.github.io/MeshCentral/meshcentral/tipsntricks/#fancy-config-editing-with-vs-code - also check this out with VSCODE
